panther_analysis_tool/main.py

```diff
@@ -1306,6 +1323,7 @@ def setup_parser() -> argparse.ArgumentParser:
     test_parser.add_argument(min_test_name, **min_test_arg)
     test_parser.add_argument(path_name, **path_arg)
     test_parser.add_argument(ignore_extra_keys_name, **ignore_extra_keys_arg)
+    test_parser.add_argument(ignore_files_name, **ignore_files_arg)
```

Comment:
wonder if we need to add this to the `publish` and `upload` parsers as well.  Can you try running through all the commands?

Reply:
Good catch, `upload` did for sure. `publish` doesn't require it as it doesn't call the `load_analysis_specs` function at all in the workflow, at least not that I could find. I did test all the commands after adding to `upload` and everything seems to work well.

Reply:
I think it might, call trace I see:

publish_release-> setup_release -> generate_release_assets -> zip_analysis -> load_analysis_specs